### ISSUE TITLE
error: command '/usr/bin/nvcc' failed with exit status 1

### ISSUE THREAD

**Issue by GabbySuwichaya** [open-mmlab/mmdetection#1374]
Thanks for your error report and we appreciate it a lot.

**Checklist**
1. I have searched related issues but cannot get the expected help.(Yes)
2. The bug has not been fixed in the latest version. (Yes)

**Describe the bug**
I have got the error:  

> error: command '/usr/bin/nvcc' failed with exit status 1

which I think is a part of ...

`/usr/include/c++/6/tuple:502:1: error: body of constexpr function ‘static constexpr bool std::_TC<<anonymous>, _Elements>::_NonNestedTuple() [with _SrcTuple = std::tuple<at::Tensor&, at::Tensor&, at::Tensor&>&&; bool <anonymous> = true; _Elements = {at::Tensor&, at::Tensor&, at::Tensor&}]’ not a return-statement
     }
 ^
error: command '/usr/bin/nvcc' failed with exit status 1`

**Reproduction**
1. What command or script did you run?
```
 python setup.py develop
```
2. Did you make any modifications on the code or config? Did you understand what you have modified?

Nop. 

3. What dataset did you use?

N\A

**My Environment**
 - OS: [e.g., Ubuntu 18.04]
 - GCC [e.g., 5.5]
 - PyTorch version [1.2.0]
- How you installed PyTorch [conda]
- GPU model [GeForce GTX 1080 Ti]
- CUDA [9.1]
- NCCL 2.4.8  

**Error traceback**
If applicable, paste the error trackback here.
```
A placeholder for trackback.
```
  
/usr/include/c++/6/type_traits:1558:8: note: provided for ‘template<class _From, class _To> struct std::is_convertible’
     struct is_convertible
        ^~~~~~~~~~~~~~
/usr/include/c++/6/tuple:502:1: error: body of constexpr function ‘static constexpr bool std::_TC<<anonymous>, _Elements>::_NonNestedTuple() [with _SrcTuple = std::tuple<at::Tensor&, at::Tensor&, at::Tensor&>&&; bool <anonymous> = true; _Elements = {at::Tensor&, at::Tensor&, at::Tensor&}]’ not a return-statement
     }
 ^
error: command '/usr/bin/nvcc' failed with exit status 1

**Comment by GabbySuwichaya** [comment]
I found the problem. The problem is the fact that I used CUDA 10.1. I solved this problem by downgrade the CUDA version to CUDA 10.0.
 
I don't know what had happen there, but this seems unrelated to Pytorch (Anyway, I am new here, so it could just be my misunderstanding).  Initially I checked if Pytorch can find GPU. It turns out that with CUDA 10.1, Pytorch do just fine. Here, my Pytorch version is 1.2
